src/features/organizations/components/SearchResults/index.tsx

```diff
@@ -0,0 +1,113 @@
+import Fuse from 'fuse.js';
+import NextLink from 'next/link';
+import { useMemo } from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+
+import Ancestors from './Ancestors';
+import ProceduralColorIcon from '../ProceduralColorIcon';
+import { TreeItemData } from '../../types';
+
+interface SearchResultsProps {
+  flatOrgData: TreeItemData[];
+  onSwitchOrg: () => void;
+  searchString: string;
+}
+
+const SearchResults = ({
+  flatOrgData,
+  onSwitchOrg,
+  searchString,
```

Comment:
I would expect something like this:

```suggestion
  matchingOrgs,
  onSwitchOrg,
```